tron/api/resource.py

```diff
@@ -395,7 +398,22 @@ def __init__(self, master_control):
 
     @AsyncResource.bounded
     def render_GET(self, request):
-        return respond(request, {'status': "I'm alive."})
+        return respond(request, {
+            'status': "I'm alive.",
+            'version': __version__,
+        })
+
+
+class MetricsResource(resource.Resource):
+
+    isLeaf = True
```

Comment:
What is this for?

Reply:
It's for twisted resources. If True, it means that there aren't any subpaths (e.g. /api/metrics/something_else), and twisted won't try to call getChild.